pytensor/graph/op.py

```diff
@@ -357,7 +358,10 @@ def grad(
         .. [1] Giles, Mike. 2008. “An Extended Collection of Matrix Derivative Results for Forward and Reverse Mode Automatic Differentiation.”
 
         """
-        raise NotImplementedError()
+        # raise NotImplementedError()
```

Comment:
This is my suggestion for a simpler solution
```suggestion
        raise NotImplementedError(f"grad not implemented for Op {self}")
```

Reply:
Also your implementation was wrong because it assumed there is a single input